perun/profile/factory.py

```diff
@@ -72,10 +77,11 @@ def update_resources(self, resource_list, resource_type='list', clear_existing_r
         elif resource_type == 'snapshots':
             # Resources are in type of [{'time': _, 'resources': []}
             for i, snapshot in enumerate(resource_list):
-                self._translate_resources(snapshot['resources'], {
-                    'snapshot': i,
-                    'time': snapshot.get('time', '0.0')}
-                )
+                self._translate_resources(snapshot['resources'],
```

Comment:
Fix: This indentation seems weird. Either keep the previous one or extract to variable.